package.json

```diff
@@ -33,20 +33,21 @@
   ],
   "homepage": "https://github.com/geostyler/geostyler",
   "scripts": {
-    "build": "npm run build:dist && npm run build:browser && npm run build:styleguide",
-    "build:dist": "tsc -p ./ && copyfiles \"./src/**/*.less\" dist --up 1 && node transform-less.js",
-    "build:styleguide": "styleguidist build",
-    "build:browser": "webpack --config browser-build.config.js",
+    "build": "npm run build-dist && npm run build-browser && npm run build-styleguide",
```

Comment:
According to your naming convention, shouldn't this rather be `build-dist:build-browser:build-styleguide`?

Reply:
nah, I'd opt for the short name